install_sct

```diff
@@ -744,3 +744,6 @@ else
 fi
 
 ) 2>&1 | tee install_sct_log.txt
+status=$?
```

Comment:
Will `$?` propagate the exit status of the subshell, or will it just propagate the exit status of the following `tee` command?

Reply:
Good catch, thanks! And as a result, TIL about `${PIPESTATUS[0]}` (see [this stackexchange question](https://stackoverflow.com/questions/1221833/pipe-output-and-capture-exit-status-in-bash#1221870)). I'll use that instead.

Reply:
Oh, good find!! :heart: